src/lib/render-lit-html.ts

```diff
@@ -228,6 +288,10 @@ export async function* renderTemplateResult(
     } else if (isElement(node)) {
       // If the element is custom, this will be the instantiated class
       let instance: LitElement | undefined = undefined;
+
+      // Whether to flush the start tag. This is neccessary if we're changing
+      // any of the attributes in the tag, so it's true for custom-elements
+      // which might reflect their own state, or any element with a binding.
```

Comment:
Possibly. That means we flush a _lot_ more string chunks, because we'd be slicing up the template string for every tag. Right now we only slice and flush up to tags with bindings or custom elements.

Reply:
Yeah ok, that makes sense.